config/crds/kudo_v1beta1_operatorversion.yaml

```diff
@@ -77,12 +78,12 @@ spec:
                 type: object
               type: array
             templates:
-              description: List of go templates YAML files that define the application
-                operator instance
+              description: Yaml captures a templated yaml list of elements that define
```

Comment:
love most of these changes... except this one.   This desc makes no sense to me.

Reply:
Yeah, second this, what about: `Templates is a list of references to YAML templates located in the templates folder and later referenced from tasks.`